examples/docker-compose.yml

```diff
@@ -28,18 +21,18 @@ services:
       args:
         COMPONENT: ${COMPONENT:-state.memory}
     environment:
-      - DAPR_COMPONENT_SOCKET_PATH=/var/run/dapr-state.store-v1-default.sock
+      - DAPR_COMPONENT_SOCKET_PATH=/var/run/${COMPONENT_NAME:-pluggable}.sock
     volumes:
       - socket:/var/run
   daprd:
     container_name: daprd-pluggable-component
-    image: "ghcr.io/mcandeia/daprd:1.9.0-pubsub-alpha-linux-amd64"
+    restart: on-failure
+    image: "ghcr.io/mcandeia/daprd:1.9.0-autoregistration-linux-amd64"
     command: "./daprd --log-level=debug --app-id pluggable-test --components-path=/components"
+    environment:
+      - DAPR_PLUGGABLE_COMPONENTS_SOCKETS_FOLDER=/var/run
     depends_on:
       - component
-      - redis
-      - kafka
-      - zookkeeper
     ports:
       - "3500:3500"
```

Comment:
Does it make sense to define ports for this example file?

Reply:
that port mapping should be used for daprd when calling http api, so I think it is necessary